1-Pflichtenheft/pflichtenheft.tex

```diff
@@ -88,6 +88,99 @@ \section{Produktumgebung}
+		\item Falls die Wahlen nicht eingeleitet sind: Das System gibt dem Benutzer eine Informationsnachricht "Wahlen haben nicht begonnen". Das System schließt seine Arbeit ab.
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Netzwerkadministratorfunktionalitaet}
+\begin{itemize}
+	\item Ziel: den Wahlprozess zu verwalten.
+	\item Subsystem: : Verwaltung der Wahlen.
+	\item Das System ueberprueft, ob der Wahlenprozess begonnen ist.	
+	\begin{enumerate}
+		\item Falls die Wahlen eingeleitet sind: Das System bietet dem Administrator die Wahl nur eine  der   folgenden Möglichkeiten:
+		\begin{enumerate}
+			\item Berechnen die aktuellen Anzahl der Votes. (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.3)
+			\item Beenden die Wahl. (Falls diese Option ausgewählt ist, gehen Sie zu 5.4)
+		\end{enumerate}
+		\item Falls die Wahlen nicht eingeleitet sind: Das System bietet dem Administrator die Wahl nur eine  der   folgenden Möglichkeiten:
+		\begin{enumerate}
+			\item Den Kandidat hinzufuegen. Falls diese Option ausgewählt ist, gehen Sie zu 5.6)
+			\item Den Kandidat loeschen Falls diese Option ausgewählt ist, gehen Sie zu 5.7)
+		\end{enumerate}
+		\item Falls die Wahlen schon abgeschlossen sind: Das System bietet dem Administrator die Wahl nur eine  der   folgenden Möglichkeiten:
+		\begin{enumerate}
+			\item Berechnen die aktuellen Anzahl der Votes. (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.3)
+			\item Neue Wahlen beginnen (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.2.2)
+		\end{enumerate}
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Zaehlen der Votes}	
+\begin{itemize}
+	\item Ziel: berechnen die Anzahl der von Waehler gegebenen Votes.
+	\item Verwaltung der Wahlen.
+	\item Das System gibt Informationen über jeden Kandidaten (Name, Nachname, Foto des Kandidaten) und die Anzahl der von jedem der Wähler erhaltenen Votes aus. Der Netzwerkadministrator nach der Überprüfung der Informationen geht zurück (siehe 5.2.1 oder 5.2.3). 
+\end{itemize}
+
+\subsection{Beendigung der Wahlen}
+\begin{itemize}
+	\item Ziel: die Wahlen beenden.
+	\item Subsystem: Verwaltung der Wahlen.
+      \item Das System gibt die Absichten des Netzwerkadministrators an, den Abschluss der Wahlen durchzuführen.
+Der Administrator stimmt zu oder lehnt ab.
+	\begin{enumerate}
+		\item Falls Netzwerkadministrator stimmt zu: Das System stoppt den Wahlenprozess. (Dann Uebergang 
+zu 5.2)
+		\item Falls Netzwerkadministrator lehnt ab: Uebergang zu 5.2
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Hinzufuegen des neuen Kandidaten}
+\begin{itemize}
+	\item Ziel: Neue Kandidat in dem Wahlprozess hinzufuegen.
+	\item Subsystem: Verwaltung der Wahlen.
+	\item Das System schlägt vor, Nachname, Vorname, Vatersnamen einzugeben und ein Foto des Kandidaten hochzuladen.
```

Comment:
Das muss abhängig davon, für welche Datenspeicherung wir uns entscheiden, geändert werden. Siehe dazu den Kommentar in Zeile 130.
Außerdem: warum wird hier zusätzlich der "Vatersname" verlangt, wenn er ansonsten keine Verwendung hat? er wird ja nicht wieder ausgegeben (siehe Zeile 130)

Reply:
gemacht